vision/unit_tests/test_module.py

```diff
@@ -363,10 +361,87 @@ def test_return(self):
         #   and arbitrary center value
         depth_image = np.zeros(IMAGE_SIZE, dtype=int)
         arbitrary_value = 300
-        roi = region_of_interest(depth_image, arbitrary_value, (arbitrary_value, arbitrary_value))
+        roi = region_of_interest(
+            depth_image, arbitrary_value, (arbitrary_value, arbitrary_value)
+        )
 
         self.assertIs(type(roi), np.ndarray)
 
 
+class TestModuleRoll(unittest.TestCase):
+    """
+    Testing module.get_module_roll for validity.
+    """
+
+    def test_params(self):
+        """
+        Verify can handle input types
+
+        Parameters
+        ----------
+        enclosing_region: ndarray
+            The depth image.
+        module_roll: np.float64
+            Calculated float angle.
+        """
+
+        IMAGE_SIZE = [1080, 1920, 3]
+
+        enclosing_region = np.zeros(IMAGE_SIZE, dtype=np.uint8)
+        module_roll = get_module_roll(enclosing_region)
+
+        self.assertIs(type(enclosing_region), np.ndarray)
+        self.assertIs(type(module_roll), np.float64)
+
+    def test_module_roll(self):
+        img_dir = os.path.join(gparent_dir, "vision_images/module/Feb29")
+        files = os.listdir(img_dir)
+
+        estimates = {
+            # FORMAT: "name_of_file" : [estimated float, (estimated center)]
+            "2020-02-29_15.36.15.167565": [20, (1180, 300)],
+            "2020-02-29_15.40.46.652448": [14, (700, 550)],
+            "2020-02-29_15.40.48.862847": [10, (1300, 630)],
+            "2020-02-29_15.40.40.444538": [5, (830, 670)],
+            "2020-02-29_15.40.54.564617": [8, (435, 650)],
+        }
+
+        for current_file in estimates.keys():
+            # Selects image
+            current_image_file = os.path.join(img_dir, current_file) + "-colorImage.jpg"
+            current_depth_file = os.path.join(img_dir, current_file) + "-depthImage.npy"
+            # Loads images
+            colorImage = cv2.imread(current_image_file)
+            depthImage = np.load(current_depth_file)
+
+            center = estimates[current_file][1]
+
+            # Sets a rough boundary around the module (angles are similar using get_module_bounds)
+            bound = [center[1] - 250, center[1] + 250, center[0] - 180, center[0] + 180]
+            colorImage = colorImage[bound[0] : bound[1], bound[2] : bound[3], :]
```

Comment:
Should use another variable name here. Since you are taking only a portion of the color image, it is strange to still call it the color image. Should be separate variable to increase readability.